src/test/java/htsjdk/tribble/index/IndexFactoryTest.java

```diff
@@ -63,7 +65,7 @@
     }
 
     @Test(dataProvider = "bedDataProvider")
-    public void testCreateLinearIndexFromBED(final File inputBEDFIle) throws Exception {
+    public void testCreateLinearIndexFromBED(final File inputBEDFIle) {
```

Comment:
Is it intentional to leave the `File` test code in place? I was expecting to see all tests updated to use `Path` instead.

Reply:
I could easily be persuaded, but the `File` overloads are still used by the rest of the world, including other (non-IndexFactory) htsjdk tests, Picard, and for the moment GATK. Until we remove or deprecate them (should we ?), these tests exercise both code paths.

Reply:
I think we should still test file paths, they get forwarded to the path versions anyway so it's not like we're losing coverage on the path implementation.  People still use the file versions so if we break them it's a problem.